src/Pages/Status/components/ServiceStatus.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import * as styles from '../css/ServiceStatus.module.css';
+
+class ServiceStatus extends PureComponent {
```

Comment:
1. This should be a functional component as it does not have state
2. Consider changing status type from number to string, define all of them here and expose them as a static prop of the functional component or as a named export